.eslintrc.js

```diff
@@ -37,7 +37,7 @@ module.exports = {
     'newline-per-chained-call': 0,
     'no-confusing-arrow': 0,
     'max-lines': ['error', { max: 300, skipBlankLines: true, skipComments: true }],
-    'no-unused-vars': 2,
+    'no-unused-vars': 1,
```

Comment:
what is the intent of changing this?

Reply:
so that the unused vars should be warning in place of error sign

Reply:
ah got it. It should be an error though right? 
If the variable is unused we don't need it in the codebase

Reply:
yeah

Reply:
but during development we should be able to differ between the actual errors and the errors from the unused vars